db/db.pl

```diff
@@ -2461,6 +2497,147 @@ sub dataNodes
         usersCreate();
         queriesCreate();
     }
+} elsif ($ARGV[1] =~ /^rollback$/) {
+
+    logmsg "It is STRONGLY recommended that you stop ALL moloch captures and viewers before proceeding.\n";
+
+    waitFor("ROLLBACK", "do you want to rollback?");
```

Comment:
* Can you verify that at least a few of the files exist before even prompting.
* Can you change to something like "do you want to rollback?  This will delete ALL data but sessions and history and restore from backups"

Reply:
Displayed the list of files used for the rollback operation.

Reply:
Updated the message according to the suggestion.